packages/experimental/Badge/src/PresenceBadge/PresenceBadge.tsx

```diff
@@ -56,3 +81,12 @@ export const PresenceBadge = compose<PresenceBadgeType>({
     };
   },
 });
+
+function isHighContrastEnabled(platformOS: string, theme: Theme) {
```

Comment:
This should be put somewhere in utils folder. And I'll be able to add macos support